com.discordapp.Discord.json

```diff
@@ -23,9 +23,10 @@
         "--talk-name=org.kde.StatusNotifierWatcher",
         "--talk-name=com.canonical.AppMenu.Registrar",
         "--talk-name=com.canonical.indicator.application",
-        "--talk-name=com.canonical.Unity.LauncherEntry",
+        "--talk-name=com.canonical.Unity",
         "--own-name=org.kde.*",
-        "--env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons"
+        "--env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons",
+        "--env=XDG_CURRENT_SESSION=KDE"
```

Comment:
Also please comment here why it is doing these workarounds.

Reply:
JSON don't supports comments as far as I know

Reply:
flatpak-builder does. `/* This is a comment */`